parser/src/main/java/io/ticofab/androidgpxparser/parser/domain/Copyright.java

```diff
@@ -0,0 +1,51 @@
+package io.ticofab.androidgpxparser.parser.domain;
+
+public class Copyright {
+
+    private final String mAuthor;
+    private final String mYear;
```

Comment:
It looks like this field could be a `DateTime` (better still, `ZonedDateTime`) instead of a `String`?

Reply:
I'm no xml schema expert, but I don't think `Integer` is correct data type here.

The GPX 1.1 schema defines the type as `xsd:gYear` and based on what I could find, it's defined either as

[ISO 8601 year with optional time zone indicator](https://www.w3.org/TR/2004/REC-xmlschema-2-20041028/datatypes.html#gYear) or as a [year with optional time zone offset](https://www.w3.org/TR/xmlschema11-2/#gYear)

Based on those definitions `2019+10` and `2019+02:00` could be valid values. That's why I think using `String` here is a safer bet.

Reply:
I see, you are right. Then the correct solution is to have this encoded as a native `DateTimeZone` object, as the object is meant to represent a date with optional time-zone information. I would suggest that UTC is assumed if no time zone information comes with it. Does it make sense to you? :)

Reply:
@ticofab, I disagree with the `DateTimeZone` type. The field doesn't represent a date, because it's only a year with an optional time zone information.

I don't think it's a good idea to allow the user to query hourOfDay() or dayOfWeek() from the parsed object, since that information doesn't exist in the original gpx data.

For that reason I still think it's best to leave it as `String` and let the library end user figure out how he/she wants use that information.